source/guides/stacks/dars.rst

```diff
@@ -172,6 +189,487 @@ DARS Usecase example
+
+   .. code-block:: xml
+
+      <configuration>
+          <property>
+              <name>mapreduce.framework.name</name>
+              <value>yarn</value>
+          </property>
+
+          <property>
+              <name>yarn.app.mapreduce.am.env</name>
+              <value>HADOOP_MAPRED_HOME=${HADOOP_HOME}</value>
+          </property>
+
+          <property>
+              <name>mapreduce.map.env</name>
+              <value>HADOOP_MAPRED_HOME=${HADOOP_HOME}</value>
+          </property>
+
+          <property>
+              <name>mapreduce.reduce.env</name>
+              <value>HADOOP_MAPRED_HOME=${HADOOP_HOME}</value>
+          </property>
+       </configuration>
+
+#. Set up the :file:`/etc/hadoop/yarn-site.xml` as follows
+
+   .. code-block:: xml
+
+      <configuration>
+          <property>
+              <name>yarn.nodemanager.aux-services</name>
+              <value>mapreduce_shuffle</value>
+          </property>
+
+          <property>
+              <name>yarn.nodemanager.auxservices.mapreduce.shuffle.class</name>
+              <value>org.apache.hadoop.mapred.ShuffleHandler</value>
+          </property>
+      </configuration>
+
+Start the Apache Hadoop daemons
+===============================
+
+#. Format the NameNode server using this command:
+
+   .. code-block:: bash
+
+      hdfs namenode -format
+
+#. Start the Apache Hadoop services
+
+   HDFS Namenode service :
+
+   .. code-block:: bash
+
+      hdfs --daemon start namenode
+
+
+   HDFS Datanode service :
+
+   .. code-block:: bash
+
+      hdfs --daemon start datanode
+
+
+   Yarn ResourceManager :
+
+   .. code-block:: bash
+
+      yarn --daemon start resourcemanager
+
+
+   Yarn NodeManager :
+
+   .. code-block:: bash
+
+      yarn --daemon start nodemanager
+
+
+   jobhistory service :
+
+   .. code-block:: bash
+
+      mapred --daemon start historyserver
+
+#. Verify the nodes are alive with this command:
+
+
+   .. code-block:: bash
+
+      yarn node -list 2
+
+   Your output will look similar to:
+
+   .. code-block:: console
+
+      Total Nodes:1
+         Node-Id             Node-State Node-Http-Address       Number-of-Running-Containers
+      <hostname>:43489            RUNNING <hostname>:8042                      0
+
+
+Run an example
+==============
+
+Apache Hadoop comes packages with a set of example  applications. In this example we will show how to use the cluster to calculate Pi. The JAR file containing the compiled class can be found on your running DARS container at :file:`/usr/share/hadoop/mapreduce/hadoop-mapreduce-examples-3.2.0.jar`
+
+
+.. code-block:: bash
+
+   hadoop jar /usr/share/hadoop/mapreduce/hadoop-mapreduce-examples-$(hadoop version | grep Hadoop | cut -d ' ' -f2).jar pi 16 100
+
+
+Deploy DARS on Kubernetes\*
+***************************
+
+Many containerized workloads are deployed in clusters managed by orchestration software like Kubernetes.
```

Comment:
```suggestion
Many containerized workloads are deployed in clusters managed by orchestration software like Kubernetes\*.
```

Reply:
The \* is applied in the section title, is it necessary here too?

Reply:
You're right. It's fine. I think I overlooked the subhead.